src/components/navBar/NavBar.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable arrow-body-style */
+// eslint-disable-next-line no-unused-vars
+import PropTypes from 'prop-types';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './navBar.module.scss';
+
+const NavBar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/calculator',
+      text: 'Calculator',
+    },
+    {
+      id: 3,
+      path: '/quote',
+      text: 'Qoute',
+    },
+  ];
+
+  const createLinks = () => {
+    return links.map((link) => (
+      <li key={link.id}>
+        <NavLink
+          to={link.path}
+          activeClassName="active-link"
+          exact
+        >
+          {link.text}
+        </NavLink>
+      </li>
+    ));
+  };
+
+  return (
+    <section className={styles.navBar}>
+      <h1>Math Magicians</h1>
+      <ul>
+        {createLinks()}
+      </ul>
+    </section>
```

Comment:
- [ ] I think `nav` is a semantically appropriate tag here than `section`. Semantic tags help provide tech support for visually impaired people and also AI-based apps. 
- [ ] I think the logo should also be part of the `nav` elements. In your design, the logo and links are not properly aligned. You can exert more effort to polish your styling to better match the wireframe design.

![image](https://user-images.githubusercontent.com/64894927/129805034-0dffa5e3-0277-4390-934d-b020fd843e23.png)

Reply:
Solved!